src/v3/test/e2e/tests/theming.spec.js

```diff
@@ -43,14 +43,15 @@ test('Theme configuration applies correctly', async (t) => {
 }).clientScripts({
   content: `
   window.additionalOptions = {
-    brandColors: {
+    brand: {
```

Comment:
Another callout that this needs to be changed since the enclosing page currently expects to pass `brandColors`